site/docs/reference/Connectors/capture-connectors/facebook-marketing.md

```diff
@@ -0,0 +1,134 @@
+| _`/custom_insights/-/fields`_ | Fields | A list of chosen fields to capture | array | `[]` |
+| _`/custom_insights/-/fields/-`_ | ValidEnums | Generic enumeration. Derive from this class to define new enumerations. | string |  |
+| _`/custom_insights/-/name`_ | Name | The name of the insight | string |  |
+| _`/custom_insights/-/start_date`_ | Start Date | The date from which you&#x27;d like to replicate data for this stream, in the format YYYY-MM-DDT00:00:00Z. | string |  |
+| _`/custom_insights/-/time_increment`_ | Time Increment | Time window in days by which to aggregate statistics. The sync will be chunked into N day intervals, where N is the number of days you specified. For example, if you set this value to 7, then all statistics will be reported as 7-day aggregates by starting from the start&#x5F;date. If the start and end dates are October 1st and October 30th, then the connector will output 5 records: 01 - 06, 07 - 13, 14 - 20, 21 - 27, and 28 - 30 (3 days only). | integer | `1` |
+| `/end_date` | End Date | The date until which you&#x27;d like to capture data, in the format YYYY-MM-DDT00:00:00Z. All data generated between start&#x5F;date and this date will be replicated. Not setting this option will result in always syncing the latest data. | string |  |
+| `/fetch_thumbnail_images` | Fetch Thumbnail Images | In each Ad Creative, fetch the thumbnail&#x5F;url and store the result in thumbnail&#x5F;data&#x5F;url | boolean | `false` |
+| `/include_deleted` | Include Deleted | Include data from deleted Campaigns, Ads, and AdSets | boolean | `false` |
+| `/page_size` | Page Size of Requests | Page size used when sending requests to Facebook API to specify number of records per page when response has pagination. Most users do not need to set this field unless they specifically need to tune the connector to address specific issues or use cases. | integer | `25` |
+| **`/start_date`** | Start Date | The date from which you&#x27;d like to begin capturing data, in the format YYYY-MM-DDT00:00:00Z. All data generated after this date will be replicated. | string | Required |
+
+#### Bindings
+
+| Property | Title | Description | Type | Required/Default |
+|---|---|---|---|---|
+| **`/stream`** | Stream | Resource of your Facebook Marketing account from which collections are captured. | string | Required |
+| **`/syncMode`** | Sync mode | Connection method. Always set to `incremental`. | string | Required |
+
+### Sample
+
+```yaml
+captures:
+  ${TENANT}/${CAPTURE_NAME}:
+    endpoint:
+      connector:
+        image: ghcr.io/estuary/airbyte-source-facebook-marketing:dev
+        config:
+            access_token: <secret>
+            business_id: 000000000000000
+            start_date: 2022-03-01T00:00:00Z
+            custom_insights:
+              name: my-custom-insight
```

Comment:
This would need a `-` here since `custom_insights` is an array, so:

```
custom_insights:
  - name: my-custom-insight
     fields: []
     breakdowns: []
     action_breakdowns: []
     start_date: ...
```